packages/client/hmi-client/src/components/widgets/tera-progress-bar.vue

```diff
@@ -0,0 +1,23 @@
+<template>
+	<div>
+		<i class="pi pi-spin pi-spinner"></i>
+		<span>{{ props.status }}...{{ props.value ? '(' + props.value + '%)' : null }}</span>
+		<ProgressBar v-if="props.value" class="progress-bar" :value="props.value"></ProgressBar>
+	</div>
+</template>
+
+<script setup lang="ts">
+import { ProgressState } from '@/types/workflow';
+import ProgressBar from 'primevue/progressbar';
+
+const props = defineProps<{
+	status: ProgressState;
+	value: number;
+}>();
+</script>
```

Comment:
This notion of `queued/running/complete` is rather specific to the workflow processes, it might make more sense to place the component there instead of under "widget".

Reply:
@mwdchang Ill move this over to workflow folder

Reply:
@mwdchang this has been moved